docker/docker-cluster-query/graphite-storage-schemas.conf

```diff
@@ -54,3 +54,4 @@
 pattern = .*
 retentions = 1s:1d
 # reorderBuffer = 20
+# reorderBufferAllowUpdate = true
```

Comment:
hmm graphite-storage-schemas.conf is for graphite, not for metrictank, so it shouldn't have anything related to metrictank in there.
maybe something for a future PR

Reply:
Yeah separate PR